docs/interfaces/hmx/leds.md

```diff
@@ -1,29 +1,36 @@
 ---
+title: LEDs (HMX)
 tags:
+  - LED
   - HMX
 ---
 
+## Overview
+
+This platform has red, green and yellow LEDs and special function leds that are external(connectors outputs).
+
+This platform also have a RGB led called indicator, that currently design(HM010-xxxXx-V6B) are hidden from view. If you can see it it is blue on u-boot and green on running linux. white flashing indicate bsp reflashing and red if something has got wrong with the flashing, see [update](../../system/update.md).
+
+All leds can be used as you like and they are just named. the yellow:power is setup from device tree with heartbeat trigger and will blink if the system is running.
```

Comment:
```suggestion
All LEDs can be used as you wish and they are just named. The yellow:power is set up from device tree with heartbeat trigger and will blink if the system is running.
```

Reply:
Does it actually blink when the system is running or is it in heartbeat mode in that case?

Reply:
LED Heartbeat Trigger found in drivers/leds/trigger/Kconfig

The configuration item CONFIG_LEDS_TRIGGER_HEARTBEAT:

    prompt: LED Heartbeat Trigger
    type: tristate
    depends on: (none)
    defined in [drivers/leds/trigger/Kconfig](https://github.com/torvalds/linux/tree/master/drivers/leds/trigger/Kconfig)
    found in Linux kernels: 3.10–3.19, 4.0–4.20, 5.0–5.19, 6.0–6.6, 6.7-rc+HEAD
    modules built: ledtrig-heartbeat, ledtrig-heartbeat

Help text

**This allows LEDs to be controlled by a CPU load average. The flash frequency is a hyperbolic function of the 1-minute load average. If unsure, say Y.**